packages/backend/config/env/development.json

```diff
@@ -12,6 +12,10 @@
       "confirmationTemplateId": "@@SENDGRID_CONFIRMATION_TEMPLATE",
       "resetPasswordTemplateId": "@@SENDGRID_RESET_PASSWORD_TEMPLATE",
       "apiKey": "@@SENDGRID_API_KEY"
+    },
+    "defaultAdmin": {
```

Comment:
Please update documentation, docker files, docker-compose and examples to be in line with these changes.